lib/spack/external/py/_path/local.py

```diff
@@ -205,14 +205,14 @@ def remove(self, rec=1, ignore_errors=False):
             if rec:
                 # force remove of readonly files on windows
                 if iswin32:
-                    self.chmod(448, rec=1) # octcal 0700
+                    self.chmod(0o700, rec=1)
                 py.error.checked_call(py.std.shutil.rmtree, self.strpath,
                     ignore_errors=ignore_errors)
             else:
                 py.error.checked_call(os.rmdir, self.strpath)
         else:
             if iswin32:
-                self.chmod(448) # octcal 0700
+                self.chmod(0o700)
```

Comment:
Ah, I had `git-blame`'d it to @tgamblin, but had missed it was a wholesale-import, not a Python3-related fix. I'll send it upstream.